src/pyodide-py/pyodide/webloop.py

```diff
@@ -223,7 +223,7 @@ def get_task_factory(self):
         return self._task_factory
 
 
-class WebLoopPolicy(asyncio.DefaultEventLoopPolicy):
+class WebLoopPolicy(asyncio.DefaultEventLoopPolicy):  # type: ignore
```

Comment:
Similarly mypy errors with a message that DefaultEventLoopPolicy is not a valid type.